packages/react-integration/cypress/integration/datalist.spec.ts

```diff
@@ -61,22 +52,6 @@ describe('Data List Compact Demo Test', () => {
     cy.get('#simple-item3').contains('Secondary content (pf-m-no-fill)');
     cy.get('#simple-item4').contains('Secondary content (pf-m-align-right pf-m-no-fill)');
   });
-
-  it('Verify rows selectable', () => {
-    cy.get('#row1.pf-m-selectable').should('be.visible');
-    cy.get('#row2.pf-m-selectable').should('be.visible');
-
-    cy.get('#row1.pf-m-selected').should('not.be.visible');
-    cy.get('#row2.pf-m-selected').should('not.be.visible');
-
-    cy.get('#row1').click();
```

Comment:
Don't we want to keep this to test the interaction?

Reply:
I missed that this test clicks rows. The `be.visible` could be changed to `exists`.

I'd argue it's still okay to remove. We need not test that classes are added or removed for simple interactions because we know our `isSelectable && styles.modifiers.selectable` logic is good! If it wasn't this test wouldn't catch it anyways because it was added at the same time.